dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ResourcesController.java

```diff
@@ -142,7 +142,7 @@ public Result createDirectory(@ApiIgnore @RequestAttribute(value = Constants.SES
         @ApiImplicitParam(name = "pid", value = "RESOURCE_PID", required = true, dataType = "Int", example = "10"),
         @ApiImplicitParam(name = "currentDir", value = "RESOURCE_CURRENTDIR", required = true, dataType = "String")
     })
-    @PostMapping()
+    @PostMapping("/create")
```

Comment:
Thanks for your contribution, this is the restful API. Please fix ui module code